src/js/components/Layer/__tests__/Layer-test.js

```diff
@@ -65,22 +65,36 @@ const TargetLayer = props => {
 describe('Layer', () => {
   beforeEach(createPortal);
   afterEach(cleanup);
-
-  ['top', 'bottom', 'left', 'right', 'start', 'end', 'center'].forEach(
-    position =>
-      test(`position ${position}`, () => {
-        render(
-          <Grommet>
-            <Layer id="position-test" position={position}>
-              This is a layer
-            </Layer>
-          </Grommet>,
-        );
-        expectPortal('position-test').toMatchSnapshot();
-      }),
+  const positions = [
+    'top',
+    'bottom',
+    'left',
+    'right',
+    'start',
+    'end',
+    'center',
+    'top-left',
+    'top-right',
+    'bottom-left',
+    'bottom-right',
+  ];
+
+  const fullOptions = [true, false, 'horizontal', 'vertical'];
+
+  positions.forEach(position =>
```

Comment:
Can we remove this test since it is being covered in the positions - full test?

Reply:
Yes, makes more sense than leaving them